src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -206,6 +209,28 @@
       OutputListLocation="$(OutDir)assemblylist.txt"
       Condition="'$(UseLegacyXunitPerfRunner)'=='true'"
      />
+     
+     <!-- For Desktop Tests, we need to generate Binding Redirects. -->
+     <ItemGroup>
+        <_FileNamesInReferenceCopyLocalPaths Include="@(ReferenceCopyLocalPaths->'%(Filename)%(Extension)')">
+          <SourcePath>%(ReferenceCopyLocalPaths.Identity)</SourcePath>
+        </_FileNamesInReferenceCopyLocalPaths>
+        <_FileNamesInTestCopyLocalByFileNameWithoutDuplicates Include="@(_TestCopyLocalByFileNameWithoutDuplicates->'%(Filename)%(Extension)')">
```

Comment:
So ReferenceCopyLocalPaths do not have SourcePath, I just wanted to standardize the metadata that I wanted in both groups.